guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,247 @@
 [id="host-substatus-overview_{context}"]
 = Host Sub-status Overview
 
-A sub-status monitors only part of a host's capabilities.
+A sub-status monitors only a part of host's capabilities.
 
-Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
-There can be more sub-statuses depending on which plugins you add to your {Project}.
+{Project} contains the following sub-statuses:
 
-The *build* sub-status is relevant for managed hosts.
+Configuration::
+This sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
++
+Possible values:
+// TODO: numbers
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Active | OK | 0
+| Pending | OK | 1
+| No changes | OK | 2
+| No reports | OK / Warning | 3
+| Out of sync | Warning | 4
+| Error | Error | 5
+|===
++
+Additional information about the sub-status values:
```

Comment:
Reading the source I get the impression that `ConfigurationStatus` doesn't have a substatus.

Reply:
I meant that Configuration status itself is a host sub-status, will try to reword to convey the message better.